en/docs/learn/configuring-email-otp.md

```diff
@@ -236,6 +237,16 @@ as explained [here](../../setup/configuring-email-sending).
                         </ul>
                     </td>
                 </tr>
+                <tr>
+                    <td><code>emailAddressRegex</code></td>
+                    <td>When the `showEmailAddressInUI` is enabled, this provides the capability to define the way the email address to be displayed in the UI. This can be configured with a proper regex pattern as required.</td>
```

Comment:
```suggestion
                    <td>When `showEmailAddressInUI` is enabled, it provides the capability to define the way the email address should be displayed in the UI. This can be configured with a proper regex pattern as required.</td>
```